_posts/blog/2017-07-18-dwh-part2-we-need-a-storage.md

```diff
@@ -0,0 +1,154 @@
+---
+layout: post
+title: "A Journey Towards a Custom Data Warehouse Solution<br/> Part 2: We Need a Storage"
+description: 
+modified:
+categories: blog
+author: robert_bordo
+excerpt: We would like to share our second part towards a custom data warehouse solution. This time it's all about selecting a proper storage.
+tags: [Data, DWH, Data Warehouse, Big Data, Data Platform, Data Lake, Storage, Redshift, S3, Snowflake]
+image:
+date: 2017-07-18T00:39:55-04:00
+---
+
+> In the beginning we created a cluster. And the cluster was without form, and void; and nulls were upon the face of the storage.
+
+As we learned in [part 1 of our series](../dwh-part1_getting_started), a data warehouse consists of several components. 
+The key component is the storage. All the others group around it. But how can one draw a decision which storage solution to take? What is out there anyway? 
+
+## Preface
+
+In a perfect world there would be only one kind of storage that fits all the needs of current DWH development and analysis. 
+But since we are not living in that kind of place, we have several options. And its number increases, the deeper one dives into the topic. 
+There seem to be solutions for every use case you can think of.
+
+That might be a good starting point. What is our most common use case? What are we going to store? And how would we like to access our data in the end?
+
+Our major source is a massive amount of log data coming from our [app](https://play.google.com/store/apps/details?id=de.axelspringer.yana). 
+Everything the user does (e.g swiping through articles, selecting categories, leaving the app) is tracked, 
+enriched with metadata (e.g. the user’s location, app version, article identifier) and stored by a third-party service in big, semi-structured log files. 
+Having only this source, a time series database like 
+[Graphite](github.com/­graphite-project/­graphite-web)
+or 
+[InfluxDB](www.influxdata.com/­time-series-platform/­influxdb) 
+could do the job. 
+But also having slow changing master data, like user profiles, article metadata and maybe even historized data, this solution would be too limited to satisfy all our current and future needs as well. 
```

Comment:
replace "historized" 
"would be too limited to satisfy all our current and future needs as well" => "would not satisfy our current and future needs"